_posts/2018-10-23-artsy-auth.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Everything You Ever Wanted To Know About Authentication at Artsy (But Didn't Know How To Ask)
+date: 2018-10-23
+categories: [tech, oauth, security, infosec, authentication]
+author: myk
+---
+
+# Everything You Ever Wanted To Know About Authentication At Artsy (But Didn't Know How To Ask)
+
+Hi! If you're reading this you're probably trying to figure out user authentication in an artsy application.
+Depending on what you're trying to do, you may need to set up [OAuth](https://oauth.net/2/) to allow users to log
+into your application by delegating authentication to gravity. Alternately, you may be working on an application
+that will only be called by other applications - in this case, you don't care about user authentication - you just
+need to establish permissions between services. In both cases you'll be working with
+[JSON Web Tokens (JWTs)](https://jwt.io/introduction/), and the difference is how the token you're looking at is
+generated.
+
+During user authentication, on login the user receives a token which explicitly grants their user account
+permission to use a specific application. For app authentication you'll create the token in advance and simply
```

Comment:
> For app authentication ...

Not sure what this means exactly – it looks like you go into more detail below, so we could mention that here? I'd also recommend taking out the word "simply" to avoid bad feelings if people find it complicated.